components/legacy_team_sidebar/legacy_team_sidebar_controller.tsx

```diff
@@ -16,62 +18,71 @@ import * as Utils from 'utils/utils.jsx';
 import SystemPermissionGate from 'components/permissions_gates/system_permission_gate';
 import Pluggable from 'plugins/pluggable';
 
-import TeamButton from './components/team_button.jsx';
+import TeamButton from './components/team_button';
 
-export function renderView(props) {
+type Actions = {
+    getTeams: Function;
```

Comment:
instead of Function, can we get a full function signature here? something like (string) => Team[]

Reply:
Same with the 2 below this one :)

Reply:
Fixed it